src/main.ts

```diff
@@ -1,7 +1,7 @@
 import * as core from '@actions/core'
 import {getInputs} from './inputs-artifacts'
 
-async function run(): Promise<void> {
+function run(): void {
```

Comment:
wait, what?

(this is not a complain) this is me currently curious into why are we removing the async here, or also wondering why at line 15 https://github.com/scality/action-artifacts/pull/62/files#diff-4fab5baaca5c14d2de62d8d2fceef376ddddcc8e9509d86cfa5643f51b89ce3dR15 we didn't had  `await run()` before.

And the question for the node beginner like myself, are we still able to execute async functions if the "main" (`run`) is no longer async?

Reply:
await can't be set at top leve at line 15l, and node complains about it, and there is no real benefit or sense to have an async function as first function called. So I removed it. Adding await in front of input.method() was breaking some tests as well.